.circleci/config.yml

```diff
@@ -15,8 +15,13 @@ jobs:
       - run: npm install
       - run: npm run build
       - run: npm run build-storybook
+      # for playwright integration tests, we want the data files we
+      # collected in the previous step
+      - attach_workspace:
+          at: /tmp/workspace
+      - run: cp -r /tmp/workspace/data/* public/data/
```

Comment:
this question is just out of curiosity: since the backend job runs before frontend, as another way of sharing data, is it possible to  `attach_workspace` in backend after we run the python script (so it should have all the files we need), then we can just run `cp -r /tmp/workspace/data/* public/data/` in the frontend job without needing to `persist_to_workspace`?

Reply:
I'm mostly just following [the approach recommended by circleci](https://circleci.com/docs/2.0/workflows/#using-workspaces-to-share-data-among-jobs). I assume that files written to an attached workspace are not themselves persisted by default otherwise they would probably have suggested doing exactly what you suggest here. :) I'm guessing on circleci's side having an explicit persist step is easier for them to implement (and arguably maybe more robust, since it's more obvious when you're writing to persistent storage).

Reply:
> lgtm! do you know why the 2 CI tests below are still hanging? I checked the CI log and looks like they already passed? https://app.circleci.com/pipelines/github/mozilla/glean-dictionary?branch=playwright-e2e

Those are some old jobs which we replaced in the circle config in this PR. I'm guessing circleci will stop looking for them once we merge this to main.

Reply:
Oh, actually just checked and we also need to update the branch protection rules: https://github.com/mozilla/glean-dictionary/settings/branch_protection_rules

I'll do that after merging this (and ci passes)